### PR TITLE
Applying Upstream Updates - add Resolve Conflicts Locally With Drupal 8

### LINKED ISSUE #912: Applying Upstream Updates Doc Update
Re: [Applying Upstream Updates](https://pantheon.io/docs/articles/sites/code/applying-upstream-updates)
## Issue Description:
Include D8 within the "[Debug Failed Merges](https://pantheon.io/docs/articles/sites/code/applying-upstream-updates/#debug-failed-merges)" section. 
## Suggested Resolution 
Add the following: 
### Resolve Conflicts Locally With Drupal 8
From within an up-to date Git clone on your local machine:
```
git pull git://github.com/pantheon-systems/drops-8.git master
# resolve conflicts
git push origin master
```
You can add the -Xtheirs flag if you want to accept all changes.

<hr>
 ^ Is `-Xtheres` still recommended?

### PR DESCRIPTION
**Pull request by rachelwhitton**
Closes #912 

- Adds new section: **Resolve Conflicts Locally With Drupal 8**

### CODE REVIEW

File: source/docs/articles/sites/code/applying-upstream-updates.md [hunk 1]
```diff
@@ -60,15 +60,24 @@ This does not solve all problems that may arise, but it should take care of most
...
 In the event the "Auto-Resolve Conflicts" option fails, the next step is to manually pull your changes in using Git, resolve the conflicts, and then push the update up to your Pantheon site.
 
+### Resolve Conflicts Locally With Drupal 8
+From within an up-to-date Git clone on your local machine:
+```bash
+git pull git://github.com/pantheon-systems/drops-8.git master
+# resolve conflicts
+git push origin master
+```
+Add the -Xtheirs flag if you want to accept all changes.
```
Comment (bmackinney) [inline source/docs/articles/sites/code/applying-upstream-updates.md]: Can we consolidate these H3's into one command sequence, preceded by a list of or dropdown for selecting the git url they need. The `-Xtheirs` flag should be in backticks. If we don't consolidate, it should be mentioned on all subsections.

**Comment by rachelwhitton** [comment]
@bmackinney let me know what you think of tabs/panels
## Active by default
screen shot 2015-09-17 at 10 20 52 am
## Panel content toggles with selected tab
screen shot 2015-09-17 at 10 24 35 am

**Comment by nataliejeremy** [comment]
+1 Nice!

**Comment by timani** [comment]
That's fancy

**Comment by rachelwhitton** [comment]
Complete in #999

**Pull request closed without merging**